appdata/il2cpp-types.h

```diff
@@ -4588,24 +4588,23 @@ namespace app
 #pragma endregion
```

Comment:
A lot of undefined structs being pointed at. I didn't mark them all.

Reply:
I don't think it's necessary to define these structs.